src/createServer.js

```diff
@@ -1,9 +1,29 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const params = new URL(`http://${req.headers.host}${req.url}`);
+    const parts = params.pathname.split('/').filter((el) => el !== '');
+
+    const query = {};
+
+    for (const qury of params.searchParams) {
+      query[qury[0]] = qury[1];
```

Comment:
the naming here (query and qury) is kinda funny, i think it could be better though